Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml

```diff
@@ -221,7 +223,7 @@
   - type: SolutionContainerVisuals
     fillBaseName: omega-
   - type: Slippery
-    paralyzeTime: 7
+    paralyzeTime: 5 # Parkstation-BetterSoap
```

Comment:
This seems like a silly change. The amount of time is still annoyingly long, so what's the difference between the annoyingly long it was and the annoyingly long it is now?